identifiers/country-kr/single_member_constituencies.csv

```diff
@@ -251,4 +251,32 @@ ocd-division/country:kr/ed:yongin_city_3,Yongin City Third
 ocd-division/country:kr/ed:yongin_city_four,Yongin City Four
 ocd-division/country:kr/ed:yongsan,Yongsan
 ocd-division/country:kr/ed:yuseong_district_1,Yuseong District First
-ocd-division/country:kr/ed:yuseong_district_2,Yuseong District Second
\ No newline at end of file
+ocd-division/country:kr/ed:yuseong_district_2,Yuseong District Second
+ocd-division/country:kr/ed:buk_district,Buk District
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_1,Buk District and Gangseo District First

```

Comment:
I think these new OCD IDs are duplicates. According to https://en.wikipedia.org/wiki/South_Korean_Legislature_Constituencies there should only be 253 districts. I took a look at a few of these and I think they already exist. Some examples (from new district => old district):

- `ocd-division/country:kr/ed:gwangju_city_buk_district_1` => `ocd-division/country:kr/ed:gwangju_buk_district_1`
- `ocd-division/country:kr/ed:buk_district_and_gangseo_district_1` => `ocd-division/country:kr/ed:busan_buk_district_and_gangseo_district_1`
- `ocd-division/country:kr/ed:seo_district` => `ocd-division/country:kr/ed:daegu_seo_district`
- `ocd-division/country:kr/ed:daejeon_city_jung_district` => `ocd-division/country:kr/ed:daejeon_jung_district`

Reply:
Thank you for your response and review. You are right, we have double checked our sources and all of the ocdids listed in this PR already exists. We'll correct it at our side. 

I believe this PR can be closed.

Reply:
Thanks for confirming, I'll close this